api/OneConfig.api

```diff
@@ -1030,6 +1030,7 @@ public class cc/polyfrost/oneconfig/hud/HUDUtils {
 
 public abstract class cc/polyfrost/oneconfig/hud/Hud {
 	protected field enabled Z
+	public field partialTicks F
```

Comment:
call it deltaTicks

Reply:
But that's not what it is, it isn't the time between ticks (a delta) it's the time since the last tick

Reply:
So it's the difference between the last time and the current one.. so a delta?

Reply:
No, it's the time since the last tick. Meaning frame 1 it might be 0.1, frame 2 0.7 and then a tick happens so frame 3 it's 0.1, while a delta is literally the time between 2 ticks

Reply:
I guess it isn't even time, it's just a number between that goes from 0 to 1 between ticks

Reply:
Oh because this isn't in a ticking context but rendering, alright, my bad

Reply:
And it’s called partialTicks litterally everywhere else (mc, essential, …) so devs are used to using that term

Reply:
> And it’s called partialTicks litterally everywhere else (mc, essential, …) so devs are used to using that term

not in fabric, and that is the new  "standard" to me

Reply:
> > And it’s called partialTicks litterally everywhere else (mc, essential, …) so devs are used to using that term
> 
> not in fabric, and that is the new "standard" to me

~~Then where fabric support? :trollface:~~

I’ll update it in a bit